module/VuFind/tests/fixtures/configs/amazoncover/config.ini

```diff
@@ -31,12 +31,10 @@ email           = support@myuniversity.edu
 title           = "Library Catalog"
 ; This is the default theme for non-mobile devices (or all devices if mobile_theme
 ; is disabled below). Available standard themes:
-;   bootstrap3 = HTML5 theme using Bootstrap 3 + jQuery libraries, with minimal styling
-;   bootprint3 = bootstrap3 theme with more attractive default styling applied
-;                (named after the earlier, now-deprecated blueprint theme)
-;   sandal     = bootstrap3 theme with a "flat" styling applied (a newer look
-;                than bootprint3).
-theme           = bootprint3
+;   bootstrap5 = HTML5 theme using Bootstrap 5 with minimal styling.
```

Comment:
Rather than updating all of these test fixtures, I wonder if we should try to reduce them to contain less information, so they only cover the things we're actually testing. What do you think? I'm open to trying to help with that if you'd like me to!

Reply:
I'd say that's a good idea but not necessarily a part of this PR. :)

Reply:
True, but it could be done separately and merged BEFORE we merge this, to the need for changes just goes away.

Reply:
True. Please go ahead if you can assist with that!

Reply:
I might have time for such a project today if other things don't come up and interrupt me. Let me know if you'd like me to put in the effort. :-)

Reply:
Done in #4207 -- even less work than I had anticipated. :-)

Reply:
Merged!